agents/summary_agent.py

```
@@ -7,23 +7,26 @@
 """
 
 import json
+import litellm
```

Ruff (0.12.2)

`litellm` imported but unused

Remove unused import: `litellm`

(F401)